git-pull-request

```diff
@@ -50,7 +50,22 @@ repo.git.remote('prune', 'origin')
 try:
   diff = repo.git.diff(local_branch, remote_ref)
 except git.exc.GitCommandError:
-  util.fatal('Remote branch %s not available.' % remote_ref)
+  push_it = util.prompt('Remote branch %s not available; attempt to push it up? [Y/n]' % remote_ref)
+  if push_it.lower() in ['y', '']:
```

Comment:
elsewhere we've allowed `yes` perhaps we should add a util for this.

Reply:
cool, done